src/test/java/com/xpandit/plugins/xrayjenkins/Utils/ProxyUtilTest.java

```diff
@@ -0,0 +1,131 @@
+package com.xpandit.plugins.xrayjenkins.Utils;
+
+import com.xpandit.xray.service.impl.XrayClientImpl;
+import com.xpandit.xray.service.impl.delegates.HttpRequestProvider;
+import hudson.ProxyConfiguration;
+import jenkins.model.Jenkins;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.powermock.reflect.Whitebox;
+
+import java.io.IOException;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.when;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({Jenkins.class, ProxyConfiguration.class})
+@PowerMockIgnore({"javax.net.ssl.*", "javax.management.*"})
+public class ProxyUtilTest {
+
+    private static final String PROXY_HOSTNAME = "whatever";
+    private static final int PROXY_PORT = 8080;
+    private static final String JIRA_DOMAIN = "localhost";
+    private static final int JIRA_PORT = 8084;
+    private static final String JIRA_HTTP_URL = "http://" + JIRA_DOMAIN + ":" + JIRA_PORT;
+    private static final String JIRA_USERNAME = "admin";
+    private static final String JIRA_PASSWORD = "admin";
+
+    @Mock
+    private Jenkins jenkins;
+
+    @Mock
+    private ProxyConfiguration proxyConfiguration;
+
+    private XrayClientImpl xrayClient;
+
+    @Before
+    public void setup() throws IOException {
+        PowerMockito.mockStatic(Jenkins.class);
+        PowerMockito.mockStatic(ProxyConfiguration.class);
+
+        Whitebox.setInternalState(jenkins, "proxy", proxyConfiguration);
+        Whitebox.setInternalState(jenkins, "proxy", proxyConfiguration);
```

Comment:
this line is cloned, can this line be removed?

Reply:
Yes, it was added by accident.